### PR TITLE
SSH and Passwd

### PR DESCRIPTION
**Pull request by SAtacker**
[asciicast](https://asciinema.org/a/430542)

### CODE REVIEW

**Comment by ArthurSonzogni** [comment]
Thanks!
Input elements seems to always appear as "Focused". I believe this was a bug I introduced recently and fixed in:
https://github.com/ArthurSonzogni/FTXUI/pull/188/commits/3657b3426aadb7867652c6c14b99fe92f67e0201
You probably want to refresh FTXUI version to the latest commit to avoid thi.

File: src/ui/panel/passwd/passwd.cpp [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+      fprintf(fp, "%s\n", new_pass.data());
+      pclose(fp);
+    });
+    page = Renderer(Container::Vertical({
+                        input_p,
+                        input_np,
+                        change_,
+                    }),
+                    [&] {
+                      if (view) {
```
Comment (ArthurSonzogni) [inline src/ui/panel/passwd/passwd.cpp:36]: I don't understand what the goal of "view" here. Could you please explain it to me? It seems this cause the layout to shift when you press enter.

**Review by ArthurSonzogni — APPROVED** [review]

**Comment by SAtacker** [comment]
In case someone wants to verify what they have typed, a small text region
is useful.

**Comment by SAtacker** [comment]
Thanks, I'll update it.

**Pull request merged**